clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -5390,6 +5390,7 @@ def AtomicCmpXchg : CIR_Op<"atomic.cmp_xchg",
                        CIR_AnyType:$desired,
                        Arg<MemOrder, "success memory order">:$succ_order,
                        Arg<MemOrder, "failure memory order">:$fail_order,
+                       OptionalAttr<I64Attr>:$alignment,
```

Comment:
Please update the assembly format below and include the alignment information. You could take the assembly format of `cir.load` as a reference.